addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3285,7 +3285,7 @@ public Action:MakeBoss(Handle:timer, any:boss)
 	}
 
 	new entity=-1;
-	while((entity=FindEntityByClassname2(entity, "tf_wearable"))!=-1)
+	while((entity=FindEntityByClassname2(entity, "tf_wear*"))!=-1)
```

Comment:
Here is a [link](https://forums.alliedmods.net/showthread.php?t=207550) which shows that the asterisk works.